internal/commands/tunnel/tunnel.go

```diff
@@ -70,13 +70,16 @@ func NewCmdTunnel() *cobra.Command {
 		NewCmdTunnelStatus(),
 	)
 
+	cmd.Flags().StringVar(&o.PrivateKeyFile, "ssh-private-key", "", "set ssh key private path")
+	cmd.Flags().StringVar(&o.PublicKeyFile, "ssh-public-key", "", "set ssh key public path")
+
 	return cmd
 }
 
 func (o *TunnelOptions) Complete(cmd *cobra.Command, args []string) error {
 	cfg, err := config.InitializeConfig()
 	if err != nil {
-		return err
+		return fmt.Errorf("can`t complete options: %w", err)
```

Comment:
backtick (`) shouldn't not be used instead of apostrophe (`'`), let's update please